### ISSUE TITLE
Add CIP data to AULCC Meeting Agenda Report (SS)(Ready on PROD)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#18422]
**Incorporate CIP infos into the AULCC Meeting Agenda**
Requested by Jaclyn during UC prioritization on 7/29/24

**Which Infos are in Scope?**
- Managed by CDS
- CIP Id Number
- CIP Project Manager & Email (If not already listed)

### Acceptance Criteria for AULCC Meeting Agenda Report (56108):

Project Details Addition:

Add the following fields under the "Project Details" section for each project in the AULCC Meeting Agenda:
Managed by CDS: Include this field to indicate if the project is managed by CDS.
CIP Id Number: Display the CIP Id Number for the project.
CIP Project Manager & Email: If not already listed, include the CIP Project Manager's name and email.
Verification:

Ensure that the Project Manager name pulled corresponds to the CIP PM Name Info field. Validate data accuracy during the report generation process.
Consistency Check:

Confirm the inclusion of these fields in all relevant sections and ensure proper formatting and placement in the report.

**Comment by TracyLinder** [comment]
@vschandramouli @rajeshlaunch this is to be included in scoping UC content.

**Comment by rajeshlaunch** [comment]
These info needs to be added in the AULCC Meeting Agenda - 56108 is the report. Add these fields under project details of each project. Also, verify the project manager name pulled as CIP PM name info field?

**Comment by rajeshlaunch** [comment]
Acceptance Criteria updated and moved to Backlog

**Comment by TracyLinder** [comment]
@Nadin-Nader this was prioritized recently as UC's #1. It's ready to go on deck.

**Comment by SubhaShroff** [comment]
### Tech Notes: 
Infos with codes
1. Managed by CDS?                  - 76362
2. CIP Id Number (if applicable) - 51515
3. CIP Project Manager               - 76360
4. CIP Project Manager Email     - 76361

**Comment by SubhaShroff** [comment]
@rajeshlaunch  @Nadin-Nader 
As per AC, CIP Project Manager - 76360, CIP Project Manager Email - 76361 should be considered from infos fields. I noticed none of the UC folders have these two info fields with values entered.
There is one UC folder(RSN-11900038), but its value looks like a number(2231.275), not a person's name.
I see in the report, currently CIP Project Manager, CIP Project Manager Email are taken from People Tab.
Please review and let me know.
Thank you.

**Comment by vschandramouli** [comment]
@SubhaShroff Since the AC says those two fields are needed if not already listed, I don't think we need to make any changes since you have found that those fields are already there. 

If there needs to be validation for the names to match the info fields, that looks like it may be a separate request and should be pursued in another ticket. 

CC @Nadin-Nader @TracyLinder

**Comment by SubhaShroff** [comment]
Thanks Vishwa.
@Nadin-Nader  This is ready for UAT on TEST.
Thank you.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  @TracyLinder 
This needs to be UATed on TEST. Thank you.

**Comment by SubhaShroff** [comment]
Update: On Friday 2/18 ROW 2pm call, Tracy has asked Isabell to UAT, in chat as she was not in the call by the time. Waiting for the feedback.

**Comment by TracyLinder** [comment]
### Update

- Ticket #21122 was submitted requesting that the project manager contact info come from the INFO tab. We think currently it comes from the people record. 
- @vschandramouli can you please reach out to Jaclyn/Izzy to confirm if they want to the data pulled from info or people record. Please also inform them that the people data appears to be more reliable.

**Comment by TracyLinder** [comment]
@SubhaShroff we'll need to stay with pulling the CIP Manager data from the info tab.

**Comment by TracyLinder** [comment]
**DTS-AMANDA Discussion:**
- Make no updates to how the current fields are mapping. 
- We are adding 4 fields and they all map to the info tab.

**Comment by SubhaShroff** [comment]
@TracyLinder @Nadin-Nader 
The report is updated. Ready for UAT. Thank you.

[AULCC Agenda Report - TEST env.pdf](https://github.com/user-attachments/files/18971978/AULCC.Agenda.Report.-.TEST.env.pdf)

**Comment by SubhaShroff** [comment]
### Update:
@Nadin-Nader This is ready on TEST for UAT. Thank you.

**Comment by BondRuss** [comment]
@Nadin-Nader Jaclyn per Jaclyn, "If the case is not a CIP, the Project Manager needs to still populate on the report. Right now it is blank because it's only pulling the CIP project manager from the INFO tab. They could replace the ROW CONTACT and rename it CIP Project Manager. So, if the case is a CIP, that field populates, otherwise it stays blank."

**Comment by BondRuss** [comment]
![Image](https://github.com/user-attachments/assets/5af10f43-9702-489a-af70-847f02f0bef3)
Here is how a non cip project looks. The PM name and email is missing because it's looking in the INFO tab. This can be a problem for non cip projects. The PM needs to show up. That's why I was recommending renaming the ROW CONTACT. We don't use it.

**Comment by BondRuss** [comment]
Sorry for all the detail but I want to make sure it's captured. I'd say at this point UAT in Test is approved for the AC items 1 & 2 and 3 & 4 need some further refinement.

**Comment by BondRuss** [comment]
[Test (90).pdf](https://github.com/user-attachments/files/19133140/Test.90.pdf)

**Comment by TracyLinder** [comment]
**ROW/DTS Review**

- If the CIP Manager name/number are empty on the info tab, the report will also be empty. An update is needed.
- If there is data in the info tab, use that data
- If not, use the data for the PM on the people tab

**Comment by TracyLinder** [comment]
**More feedback from ROW:**
I think we need to schedule a scoping meeting with UC for 18422 because Jaclyn is saying we need to show all available PM info not just the CIP on CIP and the non-CIP on non-CIP. I don't understand the benefit of showing a ton of information without sub headers defining the role and responsibility of all involved parties. 

**My Feedback:**
I've asked ROW to submit a new ticket for updating the report template so that all folder contacts (from people and info tabs) are displayed. 


@SubhaShroff please move forward with the clarifications in my comment above.

**Comment by BondRuss** [comment]
@TracyLinder and @SubhaShroff this is what UC is expecting to be done with this process per Jaclyn: 

![Image](https://github.com/user-attachments/assets/647e8394-20f9-498b-98a6-458ea6f95a7a)

**Comment by TracyLinder** [comment]
@SubhaShroff disregard my instruction and pause development on this please.

**Comment by TracyLinder** [comment]
![Image](https://github.com/user-attachments/assets/96c3b02f-51cf-4709-b43f-3abf973fd7c5)

**Comment by SubhaShroff** [comment]
@Nadin-Nader  @TracyLinder Waiting on the change expectation update. Thank you.

**Comment by BondRuss** [comment]
@SubhaShroff @TracyLinder @Nadin-Nader let's discuss this ticket next Monday (03/31) during the UC DTS meeting. 
CC: @KyleErich

**Comment by TracyLinder** [comment]
### Design decision with ROW
FYI @SubhaShroff 

![Image](https://github.com/user-attachments/assets/1b7ebce7-4c2a-4fb1-9d94-d105e6beaf33)

**Comment by SubhaShroff** [comment]
@Nadin-Nader  @TracyLinder  The report is updated as expected. Its ready for UAT on TEST. Thank you.

![Image](https://github.com/user-attachments/assets/11211153-7614-41dd-9cee-b0ca659b4344)

**Comment by SubhaShroff** [comment]
@Nadin-Nader This is up for UAT on TEST. Thank you.

**Comment by TracyLinder** [comment]
@BondRuss @KyleErich this is ready for UAT.

**Comment by BondRuss** [comment]
@TracyLinder here is what the report looks like: 

![Image](https://github.com/user-attachments/assets/ac794589-bffc-4f22-beca-99ac8b11c122)

Additionally, it appears the CIP Email Address field has been removed from the info tab. 

UAT in Test is not approved.

**Comment by SubhaShroff** [comment]
Thank you for the feedback @BondRuss.
@BondRuss @TracyLinder  @Nadin-Nader 
CIP Email Address field was missing in the info tab as Russ mentioned. CIP Project Manager Email info was created in 2024 and tested folder was from 2023. So, its missing from it. I redefaulted that folder (13217866) and gave test values for the fields that we are testing in this ticket: CIP Project Manager, CIP Project Manager Email, Managed by CDS, CIP ID#. 
After that I run the report and it looks like this. 

_**More findings for CIP Project Manager Email info**_:  I created two UC folders to make sure CIP Project Manager Email is adding with application submission, one from Amanda and another from portal - 13482304, 1348230,5 respectively. Both have CIP Project Manager field added. _But, this field didn't display on portal , not sure if that is expected_.

![Image](https://github.com/user-attachments/assets/f2b51d58-978d-49c5-924d-e5a88f663a84)

Please review. Thank you.

**Comment by TracyLinder** [comment]
@BondRuss it looks like all was taken care of when Subha re-defaulted the folder. Could you please take a look again and verify UAT?

**Comment by SubhaShroff** [comment]
@Nadin-Nader This ticket is deployed in PROD and I have verified. Ready for UAT on PROD. Thank you.

**Comment by BondRuss** [comment]
@Nadin-Nader @SubhaShroff 
CC: @KyleErich 
UAT in PROD approved per UC staff and myself. 
I will post some follow-up considerations on 22057.

**Comment by Nadin-Nader** [comment]
Dont know how this got closed but @SubhaShroff this has failed UAT on PROD

**Comment by TracyLinder** [comment]
@Nadin-Nader I think that was me, mistakenly.

**Comment by SubhaShroff** [comment]
@Nadin-Nader @KyleErich 
I adjusted the space mentioned in the feedback. Please UAT on TEST, if its as before. Thank you.

**Comment by BondRuss** [comment]
@SubhaShroff per Jaclyn, it looks good now. UAT in Test approved.

**Comment by SubhaShroff** [comment]
Thank you @BondRuss.
@Nadin-Nader SCTASK0506489 - is created for PROD manual deployment. Thanks.

**Comment by SubhaShroff** [comment]
@Nadin-Nader @BondRuss @KyleErich 
This is deployed in PROD after adjusting the space. Please review.
Thank you.

**Comment by BondRuss** [comment]
@SubhaShroff thank you! I'm going to have the UC team do some UAT, and I will update you once I hear back from them.

**Comment by BondRuss** [comment]
@SubhaShroff @Nadin-Nader 
CC: @KyleErich 
It appears the UC Tracking # is not populating correctly in PROD: 

![Image](https://github.com/user-attachments/assets/cce269ed-4317-49da-8661-535680027fc4)

**Comment by SubhaShroff** [comment]
@Nadin-Nader UC Tracking# It is fixed and PROD deployment request is created - SCTASK0508410. Thank you.

**Comment by SubhaShroff** [comment]
@Nadin-Nader @BondRuss @KyleErich 
Its corrected on PROD. Please verify. Thank you.

**Comment by TracyLinder** [comment]
PROD UAT Approved

![Image](https://github.com/user-attachments/assets/d50c0a8e-08bc-4d25-b7aa-0a5a40207546)
